api/v1alpha1/hostedcluster_types.go

```diff
@@ -443,7 +443,7 @@ type APIServerNetworking struct {
 
 // NetworkType specifies the SDN provider used for cluster networking.
 //
-// +kubebuilder:validation:Enum=OpenShiftSDN;Calico
+// +kubebuilder:validation:Enum=OpenShiftSDN;Calico;OVNKubernetes
```

Comment:
Arguably, we shouldn't validate this at all, since we could theoretically support other network providers (as long as they don't need to run on master nodes). But let's just add "Other" to this for now.